src/mame/drivers/magicard.cpp

```diff
@@ -81,6 +72,50 @@
   so... they ported the game to Impera/Funworld 8bits boards, losing part of
   graphics and sound/music quality. The new product was named "Magic Card II".
 
+  On first power up many games appear to lock with an error due to nvr not being
+  initialised. The following procedures fix this...
+
+  magicard shows STATIC MEMORY ERROR.
+  Press OWNER BOOK KEEPING button and game will continue and finally show
+  OUT OF RANGE ERROR. Press OWNER BOOK KEEPING again and game shows test menu
+  with "!PRESS HOLD4 for 5sec   TO INIT MACHINE!".  Press HOLD 4 for 5 secs and
+  after this game should start without error.
+
+  magicarda/b/w/unkte06/unkpkr_w show !!ERROR!! CALL SERVICEMAN 4
+  Press OWNER BOOK KEEPING. Game enters test Press HOLD3 for next test.
+  Select PREFERENCES I and then RAM RESET.
+  magicarda/b/unkpkr_w lock with TURN OFF!
+  Quit and restart game.
+
+  magicarde/ea/f
+  Show STATIC MEMORY ERROR.
+  Press OWNER BOOK KEEPING to get past this and again when error appears later.
+  Use HOLD3 to enter menu and CLEAR to get hidden test. Press HOLD5 for 5 secs
+  will initialise machine.
+  
+  magicarde/ea alarm when credit is added and credit is then cleared so can't be played.
+  Needs further investigation.
+  
+  Lucky7i shows !! FEHLER !! BITTE TECHNIKER RUFEN 4
+  Press OWNER BOOK Keeping. Game enters test. Press HOLD 3 (Einstellungen)
+  and then HOLD 5 (RAM RESET).
+  Machine shows "GERAET JETZT AUS. UND WIEDER EINSTECKEN!!!
+
+  magicardeb fails on OUT OF RANGE ERROR..
+  magicardec/j fail on ERROR IN SETTINGS
+
+  kajotcrd gets stuck during initialisation. Possible eeprom contents issue ?
+  dallaspk requires protection device (serial port ?)
+
+  magicle/hotslots/quingo/belslots fail on I2C BUS ERROR - protection device missing.
+
+  bigdeal0 crashes with excepiton errors in cpu core.
+
+  puzzlme - appears to work. Amount of credit for each coin input is set by values
+            in the eeprom but there doesn't appear to be any tests to set these.
+			Is something missing on this game ?  It's really poor!
```

Comment:
Please refrain from making comments about game quality, maintain a scientific approach (we are not there for making game reviews).